File: StudentList.py
student_list = [
    {
        'ID': '17021300',
        'Name': 'Nguyễn Ngọc Minh',
        'Course': 'K62-CACLC2',
        'Phone number': '0969535735',
        'Email': '17021300@vnu.edu.vn',
        'Address': 'Hanoi, Vietnam'
    },
    {
        'ID': '17021304',
        'Name': 'Nguyễn Nam',
        'Course': 'K62-CACLC2',
        'Phone number': '0969535735',
        'Email': '17021300@vnu.edu.vn',
        'Address': 'Hanoi, Vietnam'
    },
    {
        'ID': '17021306',
        'Name': 'Phạm Công Nam',
        'Course': 'K62-CACLC2',
        'Phone number': '0969535735',
        'Email': '17021302@vnu.edu.vn',
        'Address': 'Hanoi, Vietnam'
    }
]


def checkStudentExistence(ID):
    for student in student_list:
        if str(ID).strip() == student['ID']:
            return student, True
    return {}, False

File: src/Student/StudentList.java
package Student;

import java.util.ArrayList;
import java.util.List;

public class StudentList {
    private List<Student> studentList = new ArrayList<Student>();

    public StudentList() {
        studentList.add(new Student("17021300",
                "Nguyễn Ngọc Minh",
                "K62-CACLC2",
                "0969535735",
                "17021300@vnu.edu.vn",
                "Hanoi, Vietnam"));

        studentList.add(new Student("17021304",
                "Nguyễn Nam",
                "K62-CACLC2",
                "0969535735",
                "17021304@vnu.edu.vn",
                "Hanoi, Vietnam"));

        studentList.add(new Student("17021306",
                "Phạm Công Nam",
                "K62-CACLC2",
                "0969535735",
                "17021306@vnu.edu.vn",
                "Hanoi, Vietnam"));
        studentList.add(new Student("17021307",
                "Tên gì đó",
                "K62-CACLC2",
                "0969535735",
                "17021307@vnu.edu.vn",
                "Hanoi, Vietnam"));

    }

    public List<Student> getStudentList() {
        return studentList;
    }
}
